src/documents/integrationIntro.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Preferences Server Integration
+layout: default
+category: Overview
+---
+
+The First Discovery Tool is integrated with the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to save user selected preferences. The users of The First Discovery Tool are usually new to the concept of preferences that can be used to customize a user interface to match their needs. They normally don't have an existing ID with the GPII Preferences Server. This integration saves user selected preferences to the GPII Preferences Server and the GPII Preferences Server returns a token that can be kept and used to retrieved the save preferences when needed. The returned token is essentially this user's newly-created user IDs with the GPII Preferences Server.
+
+*Note*: The First Discovery Tool potentially can be integrated with any preferences servers that support the save of user preferences. This document only explains how it integrates with the GPII Preferences Server.
+
+## The First Discovery Server
+
+The First Discovery Tool makes use of [OAuth2 Client Credentials Grant API](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) provided by the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to adds preferences. [OAuth2 Client Credentials Grant API](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) requires the client ID and secret to be included in the http request for OAuth2 authentication. For the security reason, these information should only be sent between servers, instead of between browsers to the GPII Preferences Server. This is why the First Discovery Server is needed. The First Discovery Server basically sends http requests to add preferences on behalf of users of the First Discovery Tool. What it does:
```

Comment:
There are some typos. Maybe the following instead:

The First Discovery Tool makes use of the [OAuth2 Client Credentials Grant API](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) provided by the [GPII Preferences Server](https://github.com/GPII/universal/blob/master/documentation/PreferencesServer.md) to securely save preferences. The [OAuth2 Client Credentials Grant API](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) requires the client ID and secret to be included in the http request for OAuth authentication, because this sensitive information must be sent server-to-server, the First Discovery Server is used to mediate the exchange.

What it does: